packages/template-block/src/TemplateBlock.tsx

```diff
@@ -222,14 +221,20 @@ export const TemplateBlock = ({ appBridge }: BlockProps): ReactElement => {
                         </div>
                     </div>
                     {!hasPreview && isEditing && (
-                        <div className="tw-pt-2 tw-w-3/4 tw-h-[70px]">
-                            <BlockInjectButton
-                                label="Choose existing template"
-                                icon={<IconPlus24 />}
-                                withMenu={false}
-                                fillParentContainer
+                        <div className="tw-flex tw-justify-between tw-items-center tw-mt-4 tw-p-[12px] tw-pl-[16px] tw-h-[70px] tw-bg-black-0 tw-border tw-border-box-neutral tw-rounded">
```

Comment:
Always best to try and use the TW values and stay away from arbitrary values

```suggestion
                        <div className="tw-flex tw-justify-between tw-items-center tw-mt-4 tw-p-3 tw-pl-4 tw-h-[70px] tw-bg-black-0 tw-border tw-border-box-neutral tw-rounded">
```

Reply:
Thanks, makes sense! 💪